server/src/instant/util/tracer.clj

```diff
@@ -71,13 +88,13 @@
         default-attributes (cond-> @last-calculated-metrics
                              true (assoc "host.name" (config/get-hostname)
                                          "process-id" @config/process-id)
-                             thread (assoc SemanticAttributes/THREAD_NAME
+                             thread (assoc "thread.name"
                                            (.getName thread)
-                                           SemanticAttributes/THREAD_ID
+                                           "thread.id"
```

Comment:
Removed SemanticAttributes because I couldn't get it to install with the latest versions of opentelemetry.